sound/soc/sof/sof-client-probes.h

```diff
@@ -54,4 +54,17 @@ struct sof_probes_ipc_ops {
 };
 
 extern const struct sof_probes_ipc_ops ipc3_probe_ops;
+extern const struct sof_probes_ipc_ops ipc4_probe_ops;
+
+struct sof_probes_priv {
+	struct dentry *dfs_points;
+	struct dentry *dfs_points_remove;
+	u32 extractor_stream_tag;
+	struct snd_soc_card card;
+	void *ipc_priv;
```

Comment:
it would be nice to document now these members.

Reply:
It sure would, but I am only the mover guy :). I try to extract the information from some where.